Firestore/Protos/CMakeLists.txt

```diff
@@ -169,17 +170,18 @@ add_custom_target(
   VERBATIM
 
   COMMAND
+    cmake -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/generated_protos
     mkdir -p "${CMAKE_CURRENT_BINARY_DIR}/generated_protos"
   COMMAND
     ./build-protos.sh
       "$<TARGET_FILE:protoc>"
       "$<TARGET_FILE:grpc_objective_c_plugin>"
       "${FIREBASE_BINARY_DIR}/external/src/grpc/third_party/protobuf/python"
-      "${NANOPB_GENERATOR_DIR}"
+      "${NANOPB_DIR}/generator"
```

Comment:
(this was the line I was referring to in the previous commit. i.e. remove this line.)

Reply:
Yep. Done.